VideoLocker/src/main/java/org/edx/mobile/user/Account.java

```diff
@@ -0,0 +1,201 @@
+package org.edx.mobile.user;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+public class Account {
+    @SerializedName("username")
+    @NonNull
```

Comment:
Hmm, while searching around on this subject, I came across [gson-null-safe][], which seems to be mostly what we were looking for. Not sure about the performance of it though …

[gson-null-safe]: https://github.com/khernyo/gson-null-safe